MarsTest/Utilities/CommonDriver.cs

```diff
@@ -6,20 +6,19 @@ namespace MarsTest.Utilities
 {
     public class CommonDriver
     {
-        public IWebDriver driver;
+        public static IWebDriver driver;
+        
+        //public void Open()
+        //{
+        //    driver = new ChromeDriver();
+        //    driver.Manage().Window.Maximize();
```

Comment:
why is this commented out?

Reply:
so can you please removed commented code out as you should never check commented code as this is seen as bad practice. Over all your other changes look fine, once you remove the commented out code will approve

Reply:
I was doing trial and error method, will delete all uncommented codes and folders not in use